lib/taurus/qt/qtgui/taurusgui/res/setup.template

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+"""
+This file was autogenerated by taurusgui --new-gui.
+"""
+
+from setuptools import setup, find_packages
+
+# version (bumpversion style)
+__version = '0.1.0'
+
+# Example of classifier
+classifiers = [
+    'Development Status :: 3 - Alpha',
+    'Intended Audience :: End Users/Desktop',
+    'Topic :: Scientific/Engineering',
+    'Environment :: X11 Applications :: Qt',
+    'Programming Language :: Python :: 2.7'
+]
+
+_entry_points = {{
+    "gui_scripts": ["{name}=tgconf_{name}:run"]
+}}
+
+setup(
+    name="tgconf_{name}",
+    version=__version,
+    include_package_data=True,
+    packages=find_packages(),
+    entry_points=_entry_points,
+    url='http://www.taurus-scada.org/en/latest/',
+    keywords='APP',
+    description='{name} Taurus GUI',
+    requires=['setuptools (>=1.1)'],
+    install_requires=['taurus'],
```

Comment:
here we could use 'taurus[taurus-qt]' to be more precise about the dependencies (assuming that PR #479 is merged)